src/test/integration/integrationtests.py

```diff
@@ -144,6 +147,136 @@ def test_jobs_by_osu_id(self):
         self.assertEqual(
             utils.get_person_by_osu_id(not_valid_osu_id).status_code, 404)
 
+    def test_post_job_by_osu_id(self):
+        test_job = deepcopy(valid_job_body)
+
+        # expect 404 if osuID is not valid
+        self.assertEqual(
+            utils.post_job_by_osu_id(not_valid_osu_id, {}).status_code, 404)
+
+        # expect 400 if:
+
+        # POST body is empty
+        no_body = utils.post_job_by_osu_id(osu_id, {})
+        self.assertEqual(no_body.status_code, 400)
+        self.assertEqual(len(no_body.json()), 1)
+        self.assertIn("No job object provided",
+                      no_body.json()[0]["developerMessage"])
+
+        # dates are not in ISO8601 format
+        self.validate_bad_job_post(test_job, "effectiveDate", "badDate",
+                                   "Make sure dates are in ISO8601 format")
+
+        # required field is missing
+        required_fields = ["positionNumber", "hoursPerPay", "assignmentSalary",
```

Comment:
These should be the required fields https://github.com/osu-mist/persons-api/blob/076f08aaecc8491d578834aa0a6e469023331807/src/main/groovy/edu/oregonstate/mist/personsapi/PersonsResource.groovy#L251-L255